src/snowflake/snowpark/modin/plugin/_internal/ordered_dataframe.py

```diff
@@ -1672,9 +1673,22 @@ def align(
         elif how == "left":
             filter_expression = filter_expression & left_row_pos.is_not_null()
             select_list = result_projected_column_snowflake_quoted_identifiers
-        else:  # outer
+        # elif how == "right":
```

Comment:
Why is `right` removed here?

Reply:
I had added this but how='right' for ordered dataframe isn't being implemented in this PR since it would change the implementation of qc.align a bit. I should remove these commented lines